projects/plugins/protect/src/js/components/progress-bar/index.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+/**
+ * Internal dependencies
+ */
+import React from 'react';
+import styles from './style.module.scss';
+
+/**
+ * Progress Bar component
+ *
+ * @param {object} props           - Component props
+ * @param {string} props.className - Additional classnames
+ * @param {number} props.total     - Total integer
+ * @param {number} props.value     - Progress integer
+ * @returns {object}                    ProgressBar react component.
+ */
+const ProgressBar = ( { className, total = 100, value } ) => {
+	if ( value == null ) {
+		return null;
+	}
+
+	// The percentage should not be allowed to be more than 100
+	const progress = Math.min( ( value / total ) * 100, 100 );
+
+	const style = {
+		width: `${ progress }%`,
+	};
+
+	return (
+		<div className={ classnames( className, styles[ 'progress-wrapper' ] ) }>
+			<div
+				aria-valuemax={ total }
+				aria-valuemin={ 0 }
+				aria-valuenow={ Math.min( value, total ) }
+				className={ styles.progress }
+				role="progressbar"
+				style={ style }
+			></div>
+			<p className={ styles[ 'progress-percent' ] }>{ `${ progress }%` }</p>
```

Comment:
When the `<p>` is in the same container as the "progressbar" `<div>`, it takes up some of the horizontal space and the percentage width on the progress bar will not be accurate.

We should have the progress bar have a full wrapper div to itself :+1:

Reply:
Addressed this via https://github.com/Automattic/jetpack/pull/27171/commits/1157a8b8a083639cd8839623adb0490c2de3fef8